t/schema/Schedule-1.yaml

```diff
@@ -44,7 +44,7 @@ properties:
       # Ideally we should load includes before validating
       # but since test_data can contain pretty much every structure
       # we can leave it like that for now
-      "^\\$include$":
+      "^<<$":
```

Comment:
Hmm, do we need this now at all? As merge keys is part of the standard, basically.

Reply:
We just discussed that in Chat.
Currently `tools/test_yaml_valid` is just using `YAML::PP::LoadFile` without doing any merges or includes.
So we need the `<<` in the schema.
We could of course also enable merges and includes in the validation, but then we should be able to define what kind of structures the test_data can have.

Reply:
Right, but as it seems not to much to change the tests, so that we load all merges and includes and do not have special case in the schema for it. @OleksandrOrlov in case it's not that easy as I think, then please file a follow up ticket to allow merge keys and includes for any part of yaml, so we could use aliases too.

Reply:
Removed that part and replaced with the correct constructor call with 'include' and 'merge' schemas. Now it works for all the cases with import and merge, regardless of its location.